### DISCUSSION TITLE
are json references supported ?

### DISCUSSION THREAD

**Discussion by prudhomm** [nlohmann/json#2486] [Ideas]
Hi

It seems that there are discussion on JSON references e.g. [1]
I haven't found anything in the library ? Is it supported ? If not, is it planned to be supported ?

my use case is that I define various configurations with common options that I would like to share through json references

1. https://json-spec.readthedocs.io/reference.html

thanks in advance

**Comment by nlohmann** [comment]
It’s not supported, and not planned. But PRs welcome!

**Reply by prudhomm** [reply]
thanks for the answer.

ps: by the way thanks for the lib, it is very useful and well-designed.
